bundles/WebToPrintBundle/src/Installer.php

```diff
@@ -60,12 +62,13 @@ public function uninstall(): void
         // Only remove permissions. Cleanup can be done by dev or command
         $output = new ConsoleOutput(OutputInterface::VERBOSITY_NORMAL, true);
         $output->writeln([
-            "\n\n<comment>Uninstalling only removes permissions. To clean up all documents and dependencies</comment>",
+            "\n\n<comment>Uninstalling only removes permissions and doctypes. To clean up all documents and dependencies</comment>",
             '<comment>Please run <options=bold>bin/console pimcore:documents:cleanup printpage printcontainer</></comment>',
```

Comment:
but wondering, the document type clean up should be part of this console script?

```suggestion
            '<comment>Please run <options=bold>bin/console pimcore:documents:cleanup printpage printcontainer</></comment>',
```

Reply:
I thought about that too, but I decided to move it into the Installer, because there could be different SettingStore scopes. At At least that were my thoughts at the time and the type does not have to match the table name e.g. so we would need to split this in the clean up command.

Reply:
Ah ok, sounds fine to me then